mods/ra2/rules/allied-infantry.yaml

```diff
@@ -311,8 +298,7 @@ ghost:
 		Sequence: stand
 	WithInfantryBody:
 		DefaultAttackSequence: shoot
-	WithSwimSuit:
-	-TakeCover: # TODO: allow multiple sequence modifiers
+	-TakeCover:
```

Comment:
They shouldn't take cover in water afaik, i don't remember intentedly removing the comment, but it would be incorrect there. Probably should add a better comment.